languages/common-lisp/exercises/concept/basic-lists/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+Leslie the Lisp Alien needs to do some shopping. It is very important to have a shopping list. One needs to add things to it, remove things from it, and even look if something is on the list for instance.
```

Comment:
While are quite some tasks here, the key thing to keep in mind that someone proficient in Common Lisp should be able to finish this exercise in 5-10 minutes. Looking at the example implementation, I think this is actually doable with the number of tasks and their difficulty. So I'm fine with how "big" this exercise is currently.

Reply:
I really like how the tasks are written. Very engaging!

Reply:
Thanks for the feedback! Makes me feel better about how this came out.